server/src/main/java/org/cloudfoundry/identity/uaa/logging/SanitizedLogFactory.java

```diff
@@ -6,6 +6,7 @@
 /**
  * Returns Log instance that replaces \n, \r, \t with a | to prevent log forging.
  */
+@SuppressWarnings({"javasecurity:S5145", "java:S2629"}) // sanitize log messages
```

Comment:
I still have the option we should not suppress security warnings from sonar

Reply:
fixed with 16aa18b0bb9783e57f9c81fc7dc37d24eb6a98b8